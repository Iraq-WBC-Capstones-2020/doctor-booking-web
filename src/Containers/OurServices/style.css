.services-section {
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
  text-align: center;
  font-family: "Roboto";
  height: 1073px;
  background: #FAFBFD no-repeat url(./images/bg.png);
  background-position: 0 -25px;
}

.row {
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
}

.gray-line {
  width: 152px;
  height: 2;
  background: rgba(0, 0, 0, 0.5);
  border-radius: 5px;
}

.head {
  margin-top: 43px;
  width: 738px;
}

.head h1 {
  color: #026FC2;
  font-style: normal;
  font-weight: 500;
  font-size: 36px;
  line-height: 56px;
  mix-blend-mode: normal;
}

.head p {
  margin-top: 47.26px;
  font-size: 18px;
  color: #7D7987;
  font-weight: normal;
}

.card-group {
  margin: 50px 0 0 0;
  width: 898.42px;
}

.card {
  text-align: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  border-radius: 20px !important;
  -webkit-box-shadow: 10px 40px 50px rgba(229, 233, 246, 0.4);
          box-shadow: 10px 40px 50px rgba(229, 233, 246, 0.4);
}

.card img {
  margin-top: 29px;
  width: 55.81px;
  height: 54px;
}

.card .h5 {
  margin-bottom: 7px;
  font-weight: bold;
  font-size: 24px;
}

.card p {
  height: 112px;
  width: 205px;
  font-weight: normal;
  font-size: 16px;
  line-height: 28px;
  color: #7D7987;
}
/*# sourceMappingURL=style.css.map */