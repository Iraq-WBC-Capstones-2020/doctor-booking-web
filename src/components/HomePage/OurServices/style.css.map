{
    "version": 3,
    "mappings": "AAEA,AAAA,iBAAiB,CAAC;EAChB,UAAU,EAAE,MAAM;EAClB,aAAa,EAAE,KAAK;CACrB;;AAED,AAAA,IAAI,CAAC;EACH,eAAe,EAAE,YAAY;CAC9B;;AAED,AAAA,UAAU,CAAC;EACT,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,CAAC;EACT,UAAU,EAAE,kBAAkB;EAC9B,aAAa,EAAE,GAAG;CACnB;;AAED,AAAA,KAAK,CAAC;EACJ,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,KAAK;CAeb;;AAjBD,AAIE,KAJG,CAIH,EAAE,CAAC;EACD,KAAK,EClBA,OAAO;EDmBZ,WAAW,EAAE,GAAG;EAChB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;CAClB;;AATH,AAWE,KAXG,CAWH,CAAC,CAAC;EACA,UAAU,EAAE,OAAO;EACnB,SAAS,EAAE,IAAI;EACf,KAAK,EC9BA,OAAO;ED+BZ,WAAW,EAAE,MAAM;CACpB;;AAGH,AAAA,WAAW,CAAC;EACV,MAAM,EAAE,UAAU;CACnB;;AAED,AAAA,KAAK,CAAC;EACJ,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,MAAM;EACnB,aAAa,EAAE,eAAe;EAC9B,UAAU,EAAE,IAAI,CAAC,IAAI,CAAC,IAAI,CAAC,wBAAwB;CAuBpD;;AA3BD,AAME,KANG,CAMH,GAAG,CAAC;EACF,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,OAAO;EACd,MAAM,EAAE,IAAI;CACb;;AAVH,AAYE,KAZG,CAYH,GAAG,CAAC;EACF,aAAa,EAAE,GAAG;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,ECpDD,IAAI;EDqDZ,KAAK,EAAE,KAAK;CACb;;AAjBH,AAmBE,KAnBG,CAmBH,CAAC,CAAC;EACA,MAAM,EAAE,KAAK;EACb,KAAK,EAAE,KAAK;EACZ,WAAW,EAAE,MAAM;EACnB,SAAS,EC/DD,IAAI;EDgEZ,WAAW,EAAE,IAAI;EACjB,KAAK,EChEA,OAAO;CDiEb",
    "sources": [
        "style.scss",
        "_variables.scss"
    ],
    "names": [],
    "file": "style.css"
}