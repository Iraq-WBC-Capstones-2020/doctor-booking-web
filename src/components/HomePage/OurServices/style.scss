@import './variables';

body {
  background: no-repeat url(./images/bg.png);
  background-position: 0 -25px;
  font-family: 'Roboto';
}

.services-section {
  text-align: center;
  height: 1073px;
}

.row {
  justify-content: space-evenly;
}

.gray-line {
  width: 152px;
  height: 2;
  background: rgba(0, 0, 0, 0.5);
  border-radius: 5px;
}

.head {
  margin-top: 43px;
  width: 738px;

  h1 {
    color: $hColor;
    font-weight: 500;
    font-size: 36px;
    line-height: 56px;
  }

  p {
    margin-top: 47.26px;
    font-size: 18px;
    color: $pColor;
    font-weight: normal;
  }
}

.card-group {
  margin: 50px 0 0 0;
}

.card {
  text-align: center;
  align-items: center;
  border-radius: 20px !important;
  box-shadow: 10px 40px 50px rgba(229, 233, 246, 0.4);

  img {
    margin-top: 29px;
    width: 55.81px;
    height: 54px;
  }

  .h5 {
    margin-bottom: 7px;
    font-weight: 500;
    font-size: $hFontSize;
    color: black;
  }

  p {
    height: 112px;
    width: 205px;
    font-weight: normal;
    font-size: $pFontSize;
    line-height: 28px;
    color: $pColor;
  }
}
