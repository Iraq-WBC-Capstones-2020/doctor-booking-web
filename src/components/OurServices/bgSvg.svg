<svg width="844" height="901" viewBox="0 0 844 901" fill="none" xmlns="http://www.w3.org/2000/svg">
<g opacity="0.6" clip-path="url(#clip0)">
<path d="M-127.974 263.364C-70.3566 200.218 48.702 286.503 217.273 253.977C363.274 225.79 369.615 141.869 487.005 148.824C572.767 153.889 667.312 203.055 720.673 286.068C779.519 377.597 761.243 468.924 756.025 491.068C732.745 590.391 657.549 641.291 631.006 658.655C521.647 730.152 396.526 714.199 336.502 706.574C111.025 677.864 -133.553 519.788 -148.416 353.884C-150.383 332.164 -153.982 291.879 -127.974 263.364Z" fill="url(#paint0_linear)"/>
</g>
<defs>
<linearGradient id="paint0_linear" x1="429.808" y1="14.4358" x2="214.977" y2="653.783" gradientUnits="userSpaceOnUse">
<stop stop-color="#F3F7FF"/>
<stop offset="1" stop-color="#DEEEFE"/>
</linearGradient>
<clipPath id="clip0">
<rect width="913.628" height="608.795" fill="white" transform="translate(-6.31348) rotate(21.4558)"/>
</clipPath>
</defs>
</svg>
