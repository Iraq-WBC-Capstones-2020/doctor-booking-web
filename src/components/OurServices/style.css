body {
  background: no-repeat url(./images/bg.png);
  background-position: 0 -25px;
  font-family: 'Roboto';
}

.services-section {
  text-align: center;
  height: 1073px;
}

.row {
  -webkit-box-pack: space-evenly;
      -ms-flex-pack: space-evenly;
          justify-content: space-evenly;
}

.gray-line {
  width: 152px;
  height: 2;
  background: rgba(0, 0, 0, 0.5);
  border-radius: 5px;
}

.head {
  margin-top: 43px;
  width: 738px;
}

.head h1 {
  color: #026fc2;
  font-weight: 500;
  font-size: 36px;
  line-height: 56px;
}

.head p {
  margin-top: 47.26px;
  font-size: 18px;
  color: #7d7987;
  font-weight: normal;
}

.card-group {
  margin: 50px 0 0 0;
}

.card {
  text-align: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
  border-radius: 20px !important;
  -webkit-box-shadow: 10px 40px 50px rgba(229, 233, 246, 0.4);
          box-shadow: 10px 40px 50px rgba(229, 233, 246, 0.4);
}

.card img {
  margin-top: 29px;
  width: 55.81px;
  height: 54px;
}

.card .h5 {
  margin-bottom: 7px;
  font-weight: bold;
  font-size: 24px;
}

.card p {
  height: 112px;
  width: 205px;
  font-weight: normal;
  font-size: 16px;
  line-height: 28px;
  color: #7d7987;
}
/*# sourceMappingURL=style.css.map */