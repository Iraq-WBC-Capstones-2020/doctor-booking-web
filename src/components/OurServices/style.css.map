{
    "version": 3,
    "mappings": "AAGA,AAAA,IAAI,CAAA;EACF,UAAU,EAAE,SAAS,CAAC,oBAAoB;EAC1C,mBAAmB,EAAE,OAAO;EAC5B,WAAW,EAAE,QAAQ;CAEtB;;AAED,AAAA,iBAAiB,CAAC;EAChB,UAAU,EAAE,MAAM;EAClB,MAAM,EAAE,MAAM;CAEf;;AAED,AAAA,IAAI,CAAC;EACH,eAAe,EAAE,YAAY;CAC9B;;AAED,AAAA,UAAU,CAAC;EACT,KAAK,EAAE,KAAK;EACZ,MAAM,EAAE,CAAC;EACT,UAAU,EAAE,kBAAkB;EAC9B,aAAa,EAAE,GAAG;CACnB;;AAED,AAAA,KAAK,CAAC;EACJ,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,KAAK;CAgBb;;AAlBD,AAIE,KAJG,CAIH,EAAE,CAAC;EACD,KAAK,EC3BA,OAAO;ED4BZ,WAAW,EAAE,GAAG;EAChB,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;CAClB;;AATH,AAWE,KAXG,CAWH,CAAC,CAAC;EACA,UAAU,EAAE,OAAO;EACnB,SAAS,EAAE,IAAI;EACf,KAAK,ECvCA,OAAO;EDwCZ,WAAW,EAAE,MAAM;CACpB;;AAIH,AAAA,WAAW,CAAC;EACV,MAAM,EAAE,UAAU;CACnB;;AAED,AAAA,KAAK,CAAC;EACJ,UAAU,EAAE,MAAM;EAClB,WAAW,EAAE,MAAM;EACnB,aAAa,EAAE,eAAe;EAC9B,UAAU,EAAE,IAAI,CAAC,IAAI,CAAC,IAAI,CAAC,wBAAwB;CAwBpD;;AA5BD,AAME,KANG,CAMH,GAAG,CAAC;EACF,UAAU,EAAE,IAAI;EAChB,KAAK,EAAE,OAAO;EACd,MAAM,EAAE,IAAI;CACb;;AAVH,AAYE,KAZG,CAYH,GAAG,CAAC;EACF,aAAa,EAAE,GAAG;EAClB,WAAW,EAAE,GAAG;EAChB,SAAS,EC9DD,IAAI;ED+DZ,KAAK,EAAE,KAAK;CACb;;AAjBH,AAmBE,KAnBG,CAmBH,CAAC,CAAC;EACA,MAAM,EAAE,KAAK;EACb,KAAK,EAAE,KAAK;EACZ,WAAW,EAAE,MAAM;EACnB,SAAS,ECzED,IAAI;ED0EZ,WAAW,EAAE,IAAI;EACjB,KAAK,EC1EA,OAAO;CD2Eb",
    "sources": [
        "style.scss",
        "_variables.scss"
    ],
    "names": [],
    "file": "style.css"
}