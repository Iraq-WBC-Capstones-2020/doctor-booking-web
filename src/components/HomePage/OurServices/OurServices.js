import React from 'react';
import { Container, Card, CardGroup, Row } from 'react-bootstrap';
import Freame from './images/Frame.png';
import Freame2 from './images/Frame-1.png';
import Freame3 from './images/undraw_map_1r691.png';
import '../HomePageGlobalStyle.scss';

import './style.css';
import { useTranslation } from 'react-i18next';

function OurServices() {
  const { t } = useTranslation('validation');
  return (
    <Container className="services-section">
      <Row>
        <div className="head">
          <h1>{t('homePage.OurServices.headLine')}</h1>

          <hr className="grayLine" />

          <p>{t('homePage.OurServices.paragraph')}</p>
        </div>
      </Row>

      <Row>
        <CardGroup>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame} />
            <Card.Body>
              <Card.Title>
                {t('homePage.OurServices.card1.headLine')}
              </Card.Title>
              <Card.Text>{t('homePage.OurServices.card1.paragraph')}</Card.Text>
            </Card.Body>
          </Card>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame2} />
            <Card.Body>
              <Card.Title>
                {t('homePage.OurServices.card2.headLine')}
              </Card.Title>
              <Card.Text>{t('homePage.OurServices.card2.paragraph')}</Card.Text>
            </Card.Body>
          </Card>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame3} />
            <Card.Body>
              <Card.Title>
                {t('homePage.OurServices.card3.headLine')}
              </Card.Title>
              <Card.Text>{t('homePage.OurServices.card3.paragraph')}</Card.Text>
            </Card.Body>
          </Card>
        </CardGroup>
      </Row>
    </Container>
  );
}

export default OurServices;
