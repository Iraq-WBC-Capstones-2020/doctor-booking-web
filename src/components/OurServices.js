import React from 'react';
import { Container, Card, CardGroup, Row } from 'react-bootstrap';
import Freame from './images/Frame.png';
import Freame2 from './images/Frame-1.png';
import Freame3 from './images/undraw_map_1r69 1.png';

import './style.css';

function OurServices() {
  return (
    <Container className="services-section">
      <Row>
        <div className="head">
          <h1>Our services</h1>

          <hr className="gray-line" />

          <p>
            We provide to you the best choiches for you. Adjust it to your
            health needs and make sure your undergo treatment with our highly
            qualified doctors you can consult with us which type of service is
            suitable for your health
          </p>
        </div>
      </Row>

      <Row>
        <CardGroup>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame} />
            <Card.Body>
              <Card.Title>Search doctor</Card.Title>
              <Card.Text>
                Choose your doctor from thousands of specialist, general, and
                trusted hospitals
              </Card.Text>
            </Card.Body>
          </Card>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame2} />
            <Card.Body>
              <Card.Title>Details info</Card.Title>
              <Card.Text>
                Free consultation with our trusted doctors and get the best
                recomendations
              </Card.Text>
            </Card.Body>
          </Card>
          <Card className="border-0 m-4">
            <Card.Img variant="top" src={Freame3} />
            <Card.Body>
              <Card.Title>Clinic loation</Card.Title>
              <Card.Text>
                Ship of the imagination star stuff harvesting star light
                descended from astronomers finite but
              </Card.Text>
            </Card.Body>
          </Card>
        </CardGroup>
      </Row>
    </Container>
  );
}

export default OurServices;
