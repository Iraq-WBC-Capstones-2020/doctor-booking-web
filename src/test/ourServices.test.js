import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import OurServices from '../components/OurServices';

it('render1', () => {
  render(<OurServices />);
});

it('renders correctly', () => {
  const tree = renderer.create(<OurServices />).toJSON();
  expect(tree).toMatchSnapshot();
});
