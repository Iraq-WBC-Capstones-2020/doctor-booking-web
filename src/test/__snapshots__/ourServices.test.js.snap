// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="services-section container"
>
  <div
    className="row"
  >
    <div
      className="head"
    >
      <h1>
        Our services
      </h1>
      <hr
        className="gray-line"
      />
      <p>
        We provide to you the best choiches for you. Adjust it to your health needs and make sure your undergo treatment with our highly qualified doctors you can consult with us which type of service is suitable for your health
      </p>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="card-group"
    >
      <div
        className="border-0 m-4 card"
      >
        <img
          className="card-img-top"
          src="Frame.png"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Search doctor
          </div>
          <p
            className="card-text"
          >
            Choose your doctor from thousands of specialist, general, and trusted hospitals
          </p>
        </div>
      </div>
      <div
        className="border-0 m-4 card"
      >
        <img
          className="card-img-top"
          src="Frame-1.png"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Details info
          </div>
          <p
            className="card-text"
          >
            Free consultation with our trusted doctors and get the best recomendations
          </p>
        </div>
      </div>
      <div
        className="border-0 m-4 card"
      >
        <img
          className="card-img-top"
          src="undraw_map_1r69 1.png"
        />
        <div
          className="card-body"
        >
          <div
            className="card-title h5"
          >
            Clinic loation
          </div>
          <p
            className="card-text"
          >
            Ship of the imagination star stuff harvesting star light descended from astronomers finite but
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
